airbyte-integrations/connectors/source-bing-ads/integration_tests/configured_catalog.json

```diff
@@ -156,16 +156,6 @@
       "cursor_field": ["TimePeriod"],
       "destination_sync_mode": "append"
     },
-    {
```

Comment:
This is removed because it takes way too much time causing CAT to time out. We have this same report but aggregated weekly and monthly in the CAT suite so removing it should be fine